### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by NinoRisteski**
broken link on documentation tests, now fixed.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -365,7 +365,7 @@ For Transformers, the doctests are run on a daily basis via GitHub Actions as ca
 seen [here](https://github.com/huggingface/transformers/actions/workflows/doctests.yml).
 
 To include your example in the daily doctests, you need to add the filename that
-contains the example docstring to the [documentation_tests.txt](../utils/documentation_tests.txt).
+contains the example docstring to the [documentation_tests.txt](https://github.com/huggingface/transformers/blob/main/utils/slow_documentation_tests.txt).
```
Comment (amyeroberts) [inline docs/README.md:368]: We can get rid of this line completely, `documentation_tests.txt` has been removed and the user doesn't need to add the tests anywhere

**Review by amyeroberts — COMMENTED** [review]
Thanks for many fixes to our docs! 

For this PR, the link is out dated because the file doesn't exist since #25680. We can remove the whole section instead.

File: docs/README.md (hunk 1 above)
Comment (NinoRisteski) [reply docs/README.md:368]: It's my pleasure! Should I create another PR and delete the line?

Comment (amyeroberts) [reply docs/README.md:368]: Nope - you can keep this one, make the change and update the PR description

**Review by amyeroberts — APPROVED** [review]
Thanks for updating this! 

@ydshieh Is there any additional info about doc testing we should add / update alongside this change?

File: docs/README.md [hunk 2]
```diff
@@ -364,9 +364,6 @@ We use pytests' [doctest integration](https://docs.pytest.org/doctest.html) to v
 For Transformers, the doctests are run on a daily basis via GitHub Actions as can be 
 seen [here](https://github.com/huggingface/transformers/actions/workflows/doctests.yml).
 
-To include your example in the daily doctests, you need to add the filename that
-contains the example docstring to the [documentation_tests.txt](../utils/documentation_tests.txt).
```
Comment (ydshieh) [inline docs/README.md:368]: There is a new file named `utils/not_doctested.txt`. For a file to be doctested, it **SHOULD NOT** be in that file. We can add a sentence at this place :-)

**Comment by ydshieh** [comment]
Hi @NinoRisteski Sorry we forgot to merge this PR, and it ends up being done in another PR #26213.

We really appreciate your contribution and I want to apologize for what happens here 🙏 .

**Pull request closed without merging**